Source/Windows/ARM64EC/Module.cpp

```diff
@@ -56,6 +56,7 @@ desc: Implements the ARM64EC BT module API using FEXCore
 #include <winternl.h>
 #include <winnt.h>
 #include <wine/debug.h>
+#include <rpmalloc/rpmalloc.h>
```

Comment:
Move all this to (init,dinit)CRTThread, will also implicitly fix some bugs this would introduce with terminating different threads